tests.py

```diff
@@ -270,11 +282,12 @@ def test_get_sld_backward_compatibility(self):
     def test_get_sld_backward_compatibility_strict_and_wildcard_flags(self):
         psl = publicsuffix.PublicSuffixList()
         assert 'local' == psl.get_sld('local')
-        assert 'local' == psl.get_sld('foo.local')
+        # non-strict mode: TLD=local -> sld = foo.local
+        assert 'foo.local' == psl.get_sld('foo.local')
```

Comment:
this is logically correct, though not backward compatible. this is one of the old tests that didn't match the logic of alg.